### ISSUE TITLE
The `lsp` directory is created in every single project that I open with IDEA with this plugin installed

### ISSUE THREAD

**Issue by p-himik** [gtache/intellij-lsp#123]
It means that if I want to use this plugin and work with other people, I have to add it to `.gitignore` of every single project, which is not optimal.

Related issues: #93 and #78

**Comment by HenkPoley** [comment]
And the fun thing is, log files get created, but are empty.

**Comment by gtache** [comment]
The next release will put the `lsp` folder into the `.idea` one, which should already be in the `.gitignore` in the majority of the cases.     
Regarding the empty logs, if there are no errors, the error logs are empty obviously, and if the output logging is disabled in the settings, the output files will also be empty. I could not create them in this case for sure.

**Comment by p-himik** [comment]
> which should already be in the .gitignore in the majority of the cases.
This is not true at all. I don't have a single project where the whole `.idea` directory is in `.gitignore`. Some volatile and user-specific parts of it - yes. The rest is tracked by Git.

IMO a more proper solution would be to not store anything, unless requested by an explicit change in the *project-specific* configuration.

**Comment by p-himik** [comment]
An excerpt from the [How to manage projects under Version Control Systems page](https://intellij-support.jetbrains.com/hc/en-us/articles/206544839-How-to-manage-projects-under-Version-Control-Systems) (from the official IDEA support):

> Here is what you need to share:
> - All the files under the .idea directory in the project root except the workspace.xml, usage.statistics.xml, and tasks.xml files and the shelf directory which store user-specific settings
> - All the .iml module files that can be located in different module directories (applies to IntelliJ IDEA)
